Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py

```diff
@@ -1933,31 +1931,31 @@ def fetch_incidents_detection_alerts(client_obj, params: Dict[str, Any], start_t
 
     delayed_start_time = generate_delayed_start_time(time_window, start_time)
     fetch_detection_by_alert_state = pending_rule_or_version_id_with_alert_state.get('alert_state', '')
-
+    fetch_detection_by_list_basis = pending_rule_or_version_id_with_alert_state.get('listBasis', 'CREATED_TIME')
     # giving priority to comma separated detection ids over check box of fetch all live detections
     if not pending_rule_or_version_id_with_alert_state.get("rule_id") and \
             not detection_to_pull and not detection_to_process and not simple_backoff_rules:
         fetch_detection_by_ids = params.get('fetch_detection_by_ids') or ""
+        if not fetch_detection_by_ids and params.get('fetch_all_detections', False):
```

Comment:
Please check again, running type on `params.get('fetch_all_detections', False)` results in `<class 'bool'>`

Reply:
The #NOSONAR tags were placed to ignore the presence of 'http://' lines, which sonarqube linter considers as a vulnerability since HTTP transfers in plane text.